build/Linux+BSD/mscore.1.in

```diff
@@ -1,244 +1,531 @@
-@MAN_PORTABLE@but it runs on a wider range of distributions and does not need to be
-@MAN_PORTABLE@installed. There is an option to install it for full integration with other
-@MAN_PORTABLE@applications and the desktop environment.
-@END_section_to_only_appear_in_portable_builds@
-
-.SS Getting help
-This manual documents command line usage of
-.BR mscore@MSCORE_INSTALL_SUFFIX@ .
-For help with the full MuseScore program see:
-.RS
-.TP
-.B Online Handbook
-<https://musescore.org/handbook>
-.TP
-.B Support Forum
-<https://musescore.org/forum>
-.RE
-
-.SS Further information
-.TP
-These pages cover the topics in this manual and may be more up-to-date:
-
-<https://musescore.org/handbook/command\-line\-options\-0>
-.br
-<https://musescore.org/handbook/revert\-factory\-settings\-0>
-
-.SH OPTIONS
-A summary of options is included below. Running
-.B mscore@MSCORE_INSTALL_SUFFIX@
-without options launches the full MuseScore
-program and opens any specified files.
-
-@BEGIN_section_to_only_appear_in_portable_builds@
-@MAN_PORTABLE@.SS Special options for the portable version
-@MAN_PORTABLE@.TP
-@MAN_PORTABLE@.B \-h, \-\-help
-@MAN_PORTABLE@Lists the various command line options and their usage.
-@MAN_PORTABLE@.TP
-@MAN_PORTABLE@.B man, manual, manpage
+.\" New manual page for MuseScore 3.0
+.\" Copyright (c) 2018
+.\"	mirabilos <m@mirbsd.org>
+.\" Published under the same terms as MuseScore itself.
+.\"-
+.Dd December 18, 2018
+.Dt @MAN_MSCORE_UPPER@ 1
+.Os MuseScore
+.Sh NAME@Variables_substituted_by_CMAKE_on_installation@
+.Nm mscore@MSCORE_INSTALL_SUFFIX@ ,
+.Nm musescore@MSCORE_INSTALL_SUFFIX@
+.Nd @MUSESCORE_NAME_VERSION@ sheet music editor
+.Sh SYNOPSIS
+.Nm
+.Op Fl deFfhIiLmnOPRstvw
+.Op Fl a | Fl \-use\-audio Ar driver
+.Op Fl b | Fl \-bitrate Ar bitrate
+.Op Fl c | Fl \-config\-folder Ar pathname
+.Op Fl D | Fl \-monitor\-resolution Ar DPI
+.Op Fl E | Fl \-install\-extension Ar "extension file"
+.Op Fl j | Fl \-job Ar file.json
+.Op Fl M | Fl \-midi\-operations Ar file
+.Op Fl o | Fl \-export\-to Ar file
+.Op Fl p | Fl \-plugin Ar name
+.Op Fl r | Fl \-image\-resolution Ar DPI
+.Op Fl S | Fl \-style Ar style
+.Op Fl T | Fl \-trim\-image Ar margin
+.Op Fl x | Fl \-gui\-scaling Ar factor
+.Op Fl \-debug
+.Op Fl \-diff
+.Op Fl \-dump\-midi\-in
+.Op Fl \-dump\-midi\-out
+.Op Fl \-experimental
+.Op Fl \-export\-score\-parts
+.Op Fl \-factory\-settings
+.Op Fl \-force
+.Op Fl \-help
+.Op Fl \-layout\-debug
+.Op Fl \-load\-icons
+.Op Fl \-long\-version
+.Op Fl \-new\-score
+.Op Fl \-no\-fallback\-font
+.Op Fl \-no\-midi
+.Op Fl \-no\-synthesizer
+.Op Fl \-no\-webview
+.Op Fl \-raw\-diff
+.Op Fl \-revert\-settings
+.Op Fl \-run\-test\-script
+.Op Fl \-score\-media
+.Op Fl \-score\-mp3
+.Op Fl \-score\-parts\-pdf
+.Op Fl \-template\-mode
+.Op Fl \-test\-mode
+.Op Fl \-version
+.Op Ar
+.\" should be .Ss semantically but that renders badly
+@MAN_PORTABLE@.Sh Portable version, before installation
```

Comment:
`AppImage` rather than `Portable`(App), which is Windows

Reply:
I used the "portable" rather than "AppImage" phrasing when I wrote the original man page because I saw the AppImage as being the Linux equivalent of the Portable App, and because I thought that was more understandable to users at a time when AppImage was not a familiar term. However, now that AppImage has established itself as a popular packaging format it would probably be better to refer to it as just "MuseScore's AppImage".

Reply:
This is copied from the original manpage, but ok, changed.